qiskit/ignis/verification/entanglement/__init__.py

```diff
@@ -13,8 +13,7 @@
 # that they have been altered from the originals.
 
 
-"""
-In this package, there are three modules, linear.py, parallellize.py,
+"""In this package, there are three modules, linear.py, parallellize.py,
 and analysis.py. Once Terra completes design for circuit repository,
 it may be more suitable to move it there. The most important module,
```

Comment:
We should add a comment about this to #395 since we can do this today.

Reply:
Indeed. I think we need to open an issue about moving this from Ignis to Terra.